src/lib/proxy/proxyFetch.ts

```
@@ -387,346 +409,356 @@ async function createProxyAgent(proxyUrl: string): Promise<ProxyAgent> {
-    }
+      const agentCache: Map<string, ProxyAgent> =
+        (
+          globalThis as unknown as {
+            __NL_PROXY_AGENT_CACHE__?: Map<string, ProxyAgent>;
+          }
+        ).__NL_PROXY_AGENT_CACHE__ ??
+        ((
+          globalThis as unknown as {
+            __NL_PROXY_AGENT_CACHE__: Map<string, ProxyAgent>;
+          }
+        ).__NL_PROXY_AGENT_CACHE__ = new Map());
```

Biome (2.4.9)

[error] The assignment should not be in an expression.

(lint/suspicious/noAssignInExpressions)